CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## v1.22.0 (18/02/2025)
+
+**Common**
+* Added a CMake check to not change C++ standard if it's already set before.
+* Changed log severity on some messages to decrease log size.
+* Fixed getting started guide URL in README.md.
+* Removed propagation of the -Werror build option from LevelDB to parent projects if LevelDB is not installed.
+* Updated recommended version for Boost library to 1.82. The recommended version is downloaded and built if the library is not installed.
```

Comment:
```suggestion
* Added a CMake check to not change the C++ standard if it's already already set.
* Changed log severity on some messages to decrease log size.
* Fixed the Getting Started guide URL in README.md.
* Removed propagation of the -Werror build option from LevelDB to the parent projects if LevelDB is not installed.
* Updated the recommended Boost library version to 1.82. This version is downloaded and built if the library is not installed.
```

Reply:
applied with 'already already set' changes to 'already set'